modules/tinymce/src/themes/silver/test/ts/webdriver/editor/DisabledNestedMenuItemTest.ts

```diff
@@ -11,13 +12,19 @@ describe('webdriver.tinymce.themes.silver.editor.menubar.DisabledNestedMenuItemT
   const codeMenuItemSelector = '[role="menuitem"]:contains("Code")';
   const preferencesMenuItemSelector = '[title="Preferences"]';
   const servicesMenuItemSelector = '[title="Services"]';
+  const rightArrowSelector = '.tox-collection__item-caret svg';
+  const disabledColour = 'rgba(34, 47, 62, 0.5)';
+  const enabledColour = 'rgb(34, 47, 62)';
+  const highligthedCodeMenuSelector = '.tox-mbtn--active .tox-mbtn__select-label:contains("Code")';
+  const highligthedToolsMenuSelector = '.tox-mbtn--active .tox-mbtn__select-label:contains("Tools")';
 
   const hook = TinyHooks.bddSetupLight<Editor>({
     base_url: '/project/tinymce/js/tinymce',
+    plugins: 'code',
```

Comment:
We can't use `code` without importing the plugin... that said we don't need to include `code` here and we can use `edit` or `format` below instead of `tools`.

- [ ] Don't use the code plugin here